config/src/lib.rs

```diff
@@ -1,5 +1,5 @@
 //! Aggregate configuration for different Iroha modules.
-
+#![feature(absolute_path)]
```

Comment:
do we have to use unstable feature? It's only for `Display` implementation

Reply:
In this case, there's a good reason: `std::fs::absolute` is `unstable`, but it does exactly what we need. 

Given that we're using a `nightly` only toolchain, and this function will very likely be added without changes, I took the liberty of adding this feature. If we do move back to `stable` we might want to reconsider it, but for now it does what we need.

Reply:
> Tracking issue https://github.com/rust-lang/rust/issues/92750#issuecomment-1476829107 mentions that we can use Path::canonicalize() instead.

No because a lot of the time, we don't care if the file exists.

Reply:
I'm up for any solution which doesn't rely on introducing an unstable feature. Otherwise, we should take this up for a vote with the team as we discussed on the retro

Reply:
Let's keep it to `stable` then.